demisto_sdk/commands/download/tests/downloader_test.py

```diff
@@ -644,10 +644,14 @@ def test_merge_new_file(self, tmp_path):
             output_dir_path = f'{temp_dir}/{entity}'
             os.mkdir(output_dir_path)
             old_file_path = param['custom_content_object']['path']
-            new_file_path = f'{output_dir_path}/{os.path.basename(old_file_path)}'
+            base_name = os.path.basename(old_file_path)
+            if base_name.startswith("playbook-"):
+                base_name = base_name[(len("playbook-")):]
+            new_file_path = f'{output_dir_path}/{base_name}'
```

Comment:
if you handled it in the code, you want to test that it does not contain `playbook-` prefix. so why do you remove it here?

Reply:
right, removed it